src/Core/Infrastructure/RealTime/Repository/MetaService/DbReadMetaServiceRepository.php

```diff
@@ -42,14 +42,68 @@ public function __construct(DatabaseConnection $db)
      */
     public function findMetaServiceById(int $metaId): ?MetaService
     {
-        return null;
+        return $this->findMetaService($metaId);
     }
 
     /**
      * @inheritDoc
      */
     public function findMetaServiceByIdAndAccessGroupIds(int $metaId, array $accessGroupIds): ?MetaService
     {
+        if (empty($accessGroupIds)) {
+            return null;
+        }
+
+        $accessGroupRequest = ' INNER JOIN `:dbstg`.`centreon_acl` AS service_acl
+            ON service_acl.service_id = s.service_id AND service_acl.host_id = s.host_id
+            AND service_acl.group_id IN (' . implode(',', $accessGroupIds) . ') ';
+
+        return $this->findMetaService($metaId, $accessGroupRequest);
+    }
+
+    private function findMetaService(int $metaId, ?string $accessGroupRequest = null): ?MetaService
```

Comment:
+ missing phpdoc
```suggestion
    private function findMetaServiceById(int $metaId, ?string $accessGroupRequest = null): ?MetaService
```

Reply:
Impossible because the name has already been used on line 43.

Reply:
ok so only phpdoc :)